web-marketplace/src/components/layout/Layout.Header.tsx

```diff
@@ -135,11 +136,16 @@ export const LayoutHeader = () => {
           privActiveAccount,
           _,
           profileLink,
-          dashboardLink: '/dashboard',
+          dashboardLink: personalDashboardLink,
           address: wallet?.address,
         }),
         orgEnabled: ORG_ENABLED,
-        organizationProfile: menuOrganizationProfile,
+        organizationProfile: menuOrganizationProfile
+          ? {
+              ...menuOrganizationProfile,
+              dashboardLink: '/dashboard/organization',
+            }
```

Comment:
`menuOrganizationProfile` already has `dashboardLink` set as '/dashboard/organization' so you can just use it as is